frontend/src/utils/moduleHelpers.ts

```diff
@@ -25,24 +29,43 @@ export const getNumberOfAnsweredQuestions = (m: Module, tools: Tools) => {
   }
 
   const selectedTools = getSelectedTools(tools);
+  const allChecklistItems = getAllChecklistItems(m);
 
-  return Object.values(m.checkLists)
-    .flatMap(checkListItems =>
-      checkListItems.map(item => {
-        return getSupportedTools(item, selectedTools).length > 0;
-      }),
-    )
+  return allChecklistItems.map(checklistItem => {
+    return getSupportedTools(checklistItem, selectedTools).length > 0;
+  })
     .filter(isAnswered => isAnswered).length;
 };
 
+
 export const getNumberOfCheckListItems = (m: Module) => {
   if (!m.checkLists) {
     return 0;
   }
-
-  return Object.values(m.checkLists).flatMap(checklists => checklists).length;
+  return getAllChecklistItems(m).length;
 };
 
-// Ideally we'd store the category title in the data. This isn't ideal
 export const getCategoryName = (categoryData: ModuleCategory) =>
   Object.values(categoryData)[0].category;
+
+export const getModuleDescription = (m: Module) => { 
+  if(!m) return "";
+  let description = [m.assessmentQuestion];
+  let resources = m.resources;
+  let moduleDescription = m.guidance;
+
+  if(moduleDescription) description.push('', '#### Guidance:', '', moduleDescription);
+
+  if (resources) {
+    description.push('', '#### Resources:', '');
+    resources = resources.map(resource => `+ ${resource}`);
+    description = description.concat(resources);
```

Comment:
You can convert all the `let`s  (line 53-55) to `const` but doing this 
```
description.push(...resources.map(resource => `+ ${resource}`));
```

Reply:
Unlike Rust, you can still add elements to `const` arrays and objects so long as you don't try to reassign the identifier a new value (even if it's the same type)

Reply:
nice tip thanks!